lib/menu.rb

```diff
@@ -0,0 +1,25 @@
+class Menu
+
+
+def initialize
+	@menu = Hash.new
```

Comment:
Style/EmptyLiteral: Use hash literal `{}` instead of `Hash.new`. (https://github.com/bbatsov/ruby-style-guide#literal-array-hash)